src/global/Layouts/StickyNavigation/NavigationElement/NavigationElement.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon } from 'components/Icon';
+import { Col } from 'antd';
+import { Link } from 'react-router-dom';
+
+type NavActive = {
```

Comment:
I prefer type just because it has fewer letters. But we can start some discussion on it. I've created basic proposal for this. Please extend it if you can and if this will be finished we should start a discussion on that topic and vote what to do. Right now I'm leaving it as it is.
https://www.notion.so/appsome/type-vs-interface-57b2c4089a604cc9ac78c29a0a846396

Reply:
some reference about the differences: https://pawelgrzybek.com/typescript-interface-vs-type/

Reply:
It would be good to attach it to this notion proposal. I did it.